apps/aecore/src/aec_chain.erl

```diff
@@ -1169,6 +1172,7 @@ do_force_insert_headers_internal_1(
             {ok, NewTopBlock} =
                 do_find_highest_block_from_header_hash(NewTopHeaderHash,
                                                        NewHeadersDb, NewBlocksDb),
+            %% TODO Trigger check chain for successor.
```

Comment:
I need to code unit test that fails without this line implemented, then I need to code (or reuse) function to put here.

Reply:
I plan to open new PR (depending on this PR) for addressing this point.

Reply:
OK, and this is the second half of work